app/assets/stylesheets/semantic-ui/elements/_label.scss

```diff
@@ -1055,7 +1055,7 @@ a.ui.basic.label:hover {
   text-decoration: none;
   background: none #FFFFFF;
   color: #1e70bf;
-  box-shadow: 1px solid rgba(34, 36, 38, 0.15);
+  box-shadow: 1px solid $white;
```

Comment:
Does this rule even work? Looks like they changed to box-shadow instead of border without updating. The next line will in any case remove any box-shadow set by this rule.